src/components/pages/Settings/settings-widgets/AcademicTerm/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react';
+import DateTimePicker from 'react-datetime-picker';
+import { ListItemEditModal } from '~reusables/List/EditListItemModal';
+import { BaseBtn } from '~reusables/ui/Buttons';
+import { BoldText } from '~reusables/ui/Text';
+import { List, ListItem } from '~reusables/List/List';
+import { dateToDbFormat } from '~utils/format';
+import { dateType } from './u-academic-term';
+
+const AcademicTerm = () => {
+  const [date, setDate] = useState<{ [key: string]: Date | null }>({});
+  const [isModalOpen, setIsModalOpen] = useState<{ [key: string]: boolean }>(
+    {}
+  );
+
+  function handleDate(val: string, arg: Date | null) {
+    setDate((prevDate) => ({
+      ...prevDate,
+      [val]: arg,
+    }));
+  }
+
+  function handleModal(val: string) {
+    setIsModalOpen((prevModal) => ({
+      ...prevModal,
+      [val]: !prevModal[val],
+    }));
+  }
+
+  return (
+    <>
+      <div className="py-6 border-b border-gray-100">
+        <BoldText>Academic Term</BoldText>
+      </div>
+      <List>
+        {dateType.map((val, i) => {
```

Comment:
i noticed that dateType is an array of two values. I think it can be declared inline... unless if you're using it elsewhere